langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -33,6 +33,11 @@ public const float Infinity = 1.0/0.0;
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value
 #
+# ```ballerina
+# float f = 1.2;
+# boolean finite = f.isFinite();
```

Comment:
What will be the best option for update this example
```
1. 1.2.isFinite() 
2. (1.2).isFinite()
```

Reply:
I think we can use the variable here. This is somewhat similar to using a variable in `str.length()` vs `"abc".length()`. 

cc @mohanvive

Reply:
Hmm, yes. IMO, in this case introducing a variable gives more clarity.